organization/mission.md

```diff
@@ -12,10 +12,10 @@ To enable public-purpose software and policy that is open and collaborative.
 
 * We are a non-profit and non-partisan organization.
 * We publish under licenses approved by the [Open Source Initiative](https://opensource.org/licenses) or [Creative Commons](https://creativecommons.org/licenses/) licenses that do not limit commercial reuse.
-* We are controlled by our members, who are wholly publicly controlled organizations*.
+* We are controlled by our members, who are wholly publicly controlled organizations.*
 * We strive to keep our membership as accessible as possible to potential members, regardless of size or location.
 * We are committed to working with other organizations (commercial or non-profit) to achieve our mission.
 * We prioritize long term sustainability and growth for the benefit of our members
 * Our work is open by default and welcomes contributions.
 
-*)  In order to start as an association three initial individual founders will play the role of member until there are enough public members for the organization to sustain itself.
+&ast; In order to start as an association three initial individual founders will play the role of member until there are enough public members for the organization to sustain itself.
```

Comment:
Interesting choice to go for a HTML named entity, I however do not feel that this is understandable for those reading this document in MarkDown. I'd prefer a solution that works well in the text document as well as the redered one.

Perhaps we could: 

* use a different one like `†` or a `¹`.
* Escape the `*` with a slash before it, as `\*` should render the asterisk impotent.

(I also learned to put the asterisk like `*)` at the bottom of the page, so I was under the impression that was correct.)